vaadin-form-layout-flow-demo/src/main/java/com/vaadin/flow/component/formlayout/demo/FormLayoutView.java

```diff
@@ -329,4 +330,45 @@ private void createCompositeLayout() {
 
         addCard("Using form layout inside a composite", layout);
     }
+    
+    private void createLayoutHandleColspans() {
+        // begin-source-example
+        // source-example-heading: Handling columns and colspans in a layout
+    	FormLayout colLayout = new FormLayout();
+    	//Setting the desired responsive steps for the columns in the layout
```

Comment:
Just very small format thing, to add whitespace after the `//`
```suggestion
    	// Setting the desired responsive steps for the columns in the layout
```